### PR TITLE
Fix typos and other grammatical errors

### PR DESCRIPTION
**Pull request by BonfaceKilz**
#### Description

This PR (only touches the Preface):
- Fixes some typos
- Fixes some grammatical errors
- Improves sentence structure

### CODE REVIEW

**Review by liuxinyu95 — APPROVED** [review]

File: nat/nat-en.tex [hunk 1]
```diff
@@ -45,7 +45,7 @@ \section{The history of number}
 %\end{figure}
 \end{wrapfigure}
 
-The number emerged with human evolution. Some people believe the language was inspired by number. Our ancestors learned the numbers from the gathering and hunting activities. People need count the fruits gathered. As trading developed, people need numeral tools to handle bigger numbers than before.
+The number emerged with human evolution. Some people believe that language was inspired by numbers. Our ancestors learned the numbers from the gathering and hunting activities. People needed to count the gathered fruits. As trading developed, people needed numeric tools to handle bigger numbers than previously encountered.
```
Comment (liuxinyu95) [inline nat/nat-en.tex:48]: Only one comment: I think there is difference between **numeric tool** and **numeral tool**. Here I want to highlight it's about inventing the symbols/systems to handle numbers, like Arabic numeral, Roman numeral etc.

**Review by liuxinyu95 — APPROVED** [review]

**Review by liuxinyu95 — APPROVED** [review]

**Comment by liuxinyu95** [comment]
Thanks a lot for the contribution!
I'll first merge this PR, and leave the comment discussion about 'numeral/numeric' open.

**Comment by liuxinyu95** [comment]
Notes: the build failures were not caused by this change. I thought it was because LaTeX standalone regression. To unblock CI, I temporarily commented out the preambles in subfiles.

**Comment by BonfaceKilz** [comment]
I intend to audit this book slowly chapter by chapter :smile:

**Pull request merged**